docs/source/using-discovery/usingslurm.rst

```diff
@@ -17,7 +17,7 @@ with Slurm and be able to submit basic jobs on Discovery.
 
 .. important::
    Slurm commands have numerous options to help your jobs run efficiently by requesting specific resources. Options also usually have both short and verbose versions, such as
-   ``--nodes`` and ``-n`` (both mean the same thing). The examples in this documentation all use the
+   ``--nodes`` and ``-N`` (both mean the same thing). The examples in this documentation all use the
    verbose version of the options for clarity, but you can use the short version if you prefer. For example, ``srun -p short -N 1 -n 1`` means the exact same thing as ``srun --partition=short --nodes=1 --ntasks=1``
```

Comment:
can we add lowpriority as an option here to -p right after short? So something like:
srun -p short,lowpriority -N 1 -n 1 .....

Reply:
applies to both srun commands provided here